Swipbox_VIPER/Swipbox_VIPER/Application/AppConfiguration.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AppConfiguration.swift
+//  Swipbox_VIPER
+//
+//  Created by Muneeb Ali on 07/10/2024.
+//
+
+import Foundation
+
+final class AppConfiguration {
+    
+    static let shared = AppConfiguration()
+    
+    private init() {}
+    
+    private func fetchValue(forKey key: String, description: String) -> String {
+        guard let value = Bundle.main.object(forInfoDictionaryKey: key) as? String, !value.isEmpty else {
+            fatalError("\(description) must not be empty in plist. Key: \(key)")
+        }
+        return value
+    }
+    
+    lazy var apiBaseURL: String = fetchValue(forKey: "ApiBaseURL", description: "API Base URL")
+    lazy var imageBaseURL: String = fetchValue(forKey: "ImageBaseURL", description: "Image Base URL")
```

Comment:
are all of these being used in app?
If not, lets's remove the extra

Reply:
Yes, All of them are being used in app.